projects/libpsl/build.sh

```diff
@@ -17,6 +17,7 @@
 
 # avoid iconv() memleak on Ubuntu 16.04 image (breaks test suite)
 export ASAN_OPTIONS=detect_leaks=0
+export LD_LIBRARY_PATH=/usr/local/lib
```

Comment:
Yes of course. Sorry, didn't think about it.
Currently I am struck by a systemd/docker/runc issue on Debian unstable, so I can't test it. Hopefully my system at home didn't update yet... I'll ping you when it's done.